src/ducks/authentication/lib/client.js

```diff
@@ -76,14 +76,14 @@ export const initClient = url => {
     oauth: {
       storage: new Storage(),
       clientParams: {
-        redirectURI: 'http://localhost',
+        redirectURI: 'cozybanks://auth',
         softwareID: SOFTWARE_ID,
         clientName: `${SOFTWARE_NAME} (${getDeviceName()})`,
         softwareVersion: __APP_VERSION__,
         clientKind: 'mobile',
-        clientURI: 'https://gitlab.cozycloud.cc/labs/cozy-bank',
+        clientURI: 'https://github.com/cozy/cozy-banks',
         logoURI:
-          'https://gitlab.cozycloud.cc/labs/cozy-bank/raw/master/src/targets/favicons/favicon-32x32.png',
+          'https://raw.githubusercontent.com/cozy/cozy-banks/master/src/targets/favicons/favicon-32x32.png',
```

Comment:
It's weird to rely on github for that, no ? Maybe it should be part of the static files uploaded to downcloud.cozycloud.cc ?

Reply:
I fix it with logo from downcloud `email-assets` ;)